### ISSUE TITLE
Implementation for URP and HDRP

### ISSUE THREAD

**Issue by JetFalseowl** [clarte53/GaussianSplattingVRViewerUnity#25]
Hi! I understand that, according to you, this implementation works only with the Standard Render Pipeline? I tried to run it in URP and HDRP, but the splats do not render.
What needs to be fixed to implement shaders for URP and HDRP?

**Comment by Peamon** [comment]
Yes that right, it only work on standard pipeline. To move to URP or HRDP you need to change the post process system. I actually use "OnPostRender" and according to https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnPostRender.html you have to change that for URP or HDRP and use https://docs.unity3d.com/ScriptReference/Rendering.RenderPipelineManager.html
